aspnetcore/signalr/redis-backplane.md

```diff
@@ -0,0 +1,188 @@
+
+  * <xref:signalr/scale>
+  * [Redis documentation](https://redis.io/)
+  * [Azure Redis Cache documentation](https://docs.microsoft.com/en-us/azure/redis-cache/)
+
+::: moniker range="= aspnetcore-2.1"
+
+* In the SignalR app, install the `Microsoft.AspNetCore.SignalR.Redis` NuGet package.
+
+* In the `ConfigureServices` method, call `AddRedis` after `AddSignalR`:
+
+  ```csharp
+  services.AddSignalR().AddRedis(yourRedisConnectionString);
+  ```
+
+* Configure options as needed:
+ 
+  Most options can be set in the connection string or in the [ConfigurationOptions](https://stackexchange.github.io/StackExchange.Redis/Configuration#configuration-options) object. Options specified in `ConfigurationOptions` override the ones set in the connection string.
+
+  The following example shows how to set options in the `ConfigurationOptions` object. This example adds a channel prefix so that multiple apps can share the same Redis instance, as explained in the following step.
+
+  ```csharp
+  services.AddSignalR()
+    .AddRedis(connectionString, options => {
+        options.Configuration.ChannelPrefix = "MyApp";
+    });
+  ```
+
+  In the preceding code, `options.Configuration` is initialized with whatever was specified in the connection string.
+
+::: moniker-end
+
+::: moniker range="> aspnetcore-2.1"
+
+* In the SignalR app, install the `Microsoft.AspNetCore.SignalR.StackExchangeRedis` NuGet package.
+
+* In the `ConfigureServices` method, call `AddStackExchangeRedis` after `AddSignalR`:
+
+  ```csharp
+  services.AddSignalR().AddStackExchangeRedis(yourRedisConnectionString);
+  ```
+
+* Configure options as needed:
+ 
+  Most options can be set in the connection string or in the [ConfigurationOptions](https://stackexchange.github.io/StackExchange.Redis/Configuration#configuration-options) object. Options specified in `ConfigurationOptions` override the ones set in the connection string.
+
+  The following example shows how to set options in the `ConfigurationOptions` object. This example adds a channel prefix so that multiple apps can share the same Redis instance, as explained in the following step.
+
+  ```csharp
+  services.AddSignalR()
+    .AddRedis(connectionString, options => {
+        options.Configuration.ChannelPrefix = "MyApp";
+    });
+  ```
+
+  In the preceding code, `options.Configuration` is initialized with whatever was specified in the connection string.
+
+  For information about Redis options, see the [StackExchange Redis documentation](https://stackexchange.github.io/StackExchange.Redis/Configuration.html).
+
+::: moniker-end
+
+* If you're using one Redis server for multiple SignalR apps, use a different channel prefix for each SignalR app.
+
+  Setting a channel prefix isolates one SignalR app from others that use different channel prefixes. If you don't assign different prefixes, a message sent from one app to all of its own clients will go to all clients of all apps that use the Redis server as a backplane.
```

Comment:
Technically the apps would have to have the same hub names too, but I don't know if that's important enough to explain.

Reply:
Nah, better to just tell people to use a channel prefix any time they are putting multiple apps in a single Redis instance.

Reply:
Added 

> Another way to isolate the apps is to use different hub names.

Reply:
Oh, just saw Andrew's comment -- removing the added sentence.